python/GangaDirac/Dockerfile

```diff
@@ -1,34 +1,28 @@
-FROM alexanderrichards/dirac_ui:latest
+# FROM taking build arg requires at least
+#   Docker version 17.05.0-ce, build 89658be
+ARG ganga_version=latest
+FROM ganga-core:$ganga_version
```

Comment:
It is true however that this would require the ganga-core to be built before this one is. However I think that we will have to change the build process anyway as the automatic build is not available to organisations on dockerhub, only individuals, hence while it worked when it was under alexanderrichards/ganga it wont not it's under gangadevs/ganga-core etc.

We can use dockercloud to automate the builds in which case we will have to ensure that ganga-core goes first. However it is only free to build on their nodes while dockercloud is in beta. So I suggest that we make this mark of our build system in jenkins or travis this way we control the build order as well. We will of course be able to kick them off from pull requests just as with the testing.